### DISCUSSION TITLE
请问腾讯有项目在使用TDesign吗？

### DISCUSSION THREAD

**Discussion by ryanexo** [Tencent/tdesign-vue-next#4795] [General]
我非常看好TDesign，观察了TDesign很久，看到TDesign一步步的增加了各种开发规范，给每个组件配备了使用建议、应用场景，在issue里看到一个疑似TDesign的UI（我看到他发了figma链接）也会在github中跟进issue，真心觉得腾讯这个东西真是下功夫用心做了。

包括中后台模板，我也仔细看了TDesign和arco的，相比之下，arco基本就是在应付了事，连TDesign的一根毛都不如。
我现在遇到新人我都会推荐他们看TDesign的中后台的代码，看组件文档中的使用场景的建议，学习这些其他组件库甚至是教程上不会教的东西。

真心觉得TDesign如果照着这个方向发展下去，非常有前途。

但是月初在知乎看到有人提问TDesign相关的问题，第一个回答在文档的示例中就遇到了Form校验清除的bug（目前已修复），让我这种支持者看了很尴尬。

所以想问下，腾讯目前有没有项目已经使用了TDesign，给支持者们也吃一颗定心丸，让我们在安利的时候也更有底气，在做项目的时候也更容易去说服领导使用TDesign。

**Comment by uyarn** [comment]
刚刚看的数据，截止今天公司内使用TDesign的仓库数（不区分大仓小仓）是2700+个，因为公司没有限制react/vue2/vue3的使用，所以目前统计到的情况使用比例非常接近，可以大概预估一下vue3的使用仓库数。另外因为是公司各个业务本身有使用需求，所以现在几个技术栈按目前来说都会继续维护下去，同时为了进一步降低维护成本，我们也在做vue组件库的仓库合并。

**Reply by ryanexo** [reply]
谢谢回答
